.github/workflows/tests.yml

```diff
@@ -118,7 +118,15 @@ jobs:
         SYNAPSE_POSTGRES_HOST: localhost
         SYNAPSE_POSTGRES_USER: postgres
         SYNAPSE_POSTGRES_PASSWORD: postgres
-
+    - name: Dump logs #on failure
+      # Note: Dumps to workflow logs instead of using actions/upload-artifact
+      #       This keeps logs colocated with failing jobs
+      #if: ${{ failure() }}
```

Comment:
commented out?

Reply:
Testing to make sure it works as expected -- easier to do that than induce a failure :)